### PR TITLE
Tickets/PREOPS-4625

### PR DESCRIPTION
**Pull request by sgreenstreet**

### CODE REVIEW

File: DP03_05_Near-Earth_Objects.ipynb [hunk 1]
```diff
@@ -0,0 +1,1395 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cfc73be0",
+   "metadata": {},
+   "source": [
+    "## 1. Introduction\n",
+    "\n",
+    "_This tutorial demonstrates how to use TAP to query the MPCORB and SSObject tables. The tables are queried simultaneously and joined on their SSObjectId. The properties of near-Earth objects (NEOs; perihelia $q$<1.3 au) from the DP0.3 catalogs are examined. The NEOs in this sample are restricted to those with semimajor axes $a$<4.0 au, and eccentricities $e$<1.0. The orbital properties of NEOs are examined, including plotting the planet-crossing regions and boundaries that define the four NEO sub-populations: Amors, Apollos, Atens, and Atiras._"
```
Comment (ChristinaAdair) [inline DP03_05_Near-Earth_Objects.ipynb:74]: this is rendered as italic.  is that what we want for this section?

File: DP03_05_Near-Earth_Objects.ipynb [hunk 2]
```diff
@@ -0,0 +1,1395 @@
...
+    "Container Size: <i>medium</i> <br>\n",
+    "Targeted learning level: <i>beginner</i> <br>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9da1a210-d858-42fe-8591-570965b8be1a",
+   "metadata": {},
+   "source": [
+    "**Description:** _An introduction to near-Earth objects (NEOs) in the DP0.3 dataset, including illustrations of the four NEOs sub-populations: Amors, Apollos, Atens, and Atiras._"
```
Comment (ChristinaAdair) [inline DP03_05_Near-Earth_Objects.ipynb:22]: these are italicized, the dp02 nb are not, do we want to be consistent?

File: DP03_05_Near-Earth_Objects.ipynb [hunk 3]
```diff
@@ -0,0 +1,1395 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "749b0ddf",
+   "metadata": {},
+   "source": [
+    "<img align=\"left\" src = https://project.lsst.org/sites/default/files/Rubin-O-Logo_0.png width=250 style=\"padding: 10px\"> \n",
+    "<br><b>CET Template Notebook</b> <br>\n",
+    "Contact author(s): <i>Sarah Greenstreet</i> <br>\n",
```
Comment (ChristinaAdair) [inline DP03_05_Near-Earth_Objects.ipynb:10]: remove italics?

File: DP03_05_Near-Earth_Objects.ipynb [hunk 4]
```diff
@@ -0,0 +1,1395 @@
...
+  {
+   "cell_type": "markdown",
+   "id": "4695fb41-9d89-4d0b-af9c-b57222077199",
+   "metadata": {},
+   "source": [
+    "### 1.1 NEO Dynamical Classes\n",
+    "\n",
+    "Near-Earth objects (NEOs) are traditionally divided into four dynamical sub-populations based on their orbital parameters compared to the orbit of the Earth. The Earth's orbit is nearly circular with a semimajor axis $a$ = 1.0 au and eccentricity $e$ = 0.017. This gives the Earth a perihelion distance $q$ = 0.983 au and an aphelion distance of $Q$ = 1.017 au. The four NEO sub-populations (Amors, Apollos, Atens, and Atiras) are divided using these orbital parameters of the Earth as follows:\n",
+    "\n",
+    "**Amors** have orbits exterior to Earth's orbit with perihelia larger than Earth's aphelion and less than 1.3 au (1.017 au < $q$ < 1.3 au; i.e., Amors are always farther from the Sun than Earth).\n",
```
Comment (ChristinaAdair) [inline DP03_05_Near-Earth_Objects.ipynb:86]: consistent () around i.e. statement?  Or is it connected with the (1.017 au...)

File: DP03_05_Near-Earth_Objects.ipynb [hunk 5]
```diff
@@ -0,0 +1,1395 @@
...
+    "\n",
+    "**Apollos** are on Earth-crossing orbits with perihelia smaller than Earth's aphelion ($q$ < 1.017 au) and semimajor axes $a$ > 1.0 au (i.e., Apollos are Earth-crossers with orbits larger than Earth's orbit).\n",
+    "\n",
+    "**Atens** are on Earth-crossing orbits with aphelia larger than Earth's perihelion ($Q$ > 0.983 au) and semimajor axes $a$ < 1.0 au (i.e., Atens are Earth-crossers with orbits smaller than Earth's orbit).\n",
+    "\n",
+    "**Atiras** have orbits interior to Earth's orbit with aphelia smaller than Earth's perihelion ($Q$ < 0.983 au; i.e., Atiras are always closer to the Sun than Earth).\n",
```
Comment (ChristinaAdair) [inline DP03_05_Near-Earth_Objects.ipynb:92]: same comment as above...?

File: DP03_05_Near-Earth_Objects.ipynb [hunk 6]
```diff
@@ -0,0 +1,1395 @@
...
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "q_select = '1.3'\n",
+    "e_select = '1.0'\n",
+    "a_select = '4.0'\n",
+    "\n",
+    "query = \"\"\"\n",
```
Comment (ChristinaAdair) [inline DP03_05_Near-Earth_Objects.ipynb:444]: add line so """ is below "query" to be consistent

File: DP03_05_Near-Earth_Objects.ipynb [hunk 7]
```diff
@@ -0,0 +1,1395 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5a54f3e2-8c81-482d-a023-faa4eb895fda",
+   "metadata": {},
+   "source": [
+    "### Exercises to try:\n",
+    "\n",
+    "#### Plot the subset of Apollos that are only Venus-crossing (i.e., exclude the Earth-crossing Apollos).\n",
```
Comment (ChristinaAdair) [inline DP03_05_Near-Earth_Objects.ipynb:1355]: Do we want these to be in normal format (not bold?)

File: DP03_05_Near-Earth_Objects.ipynb (hunk 1 above)
Comment (sgreenstreet) [reply DP03_05_Near-Earth_Objects.ipynb:74]: Italics removed.

File: DP03_05_Near-Earth_Objects.ipynb (hunk 2 above)
Comment (sgreenstreet) [reply DP03_05_Near-Earth_Objects.ipynb:22]: Italics removed for consistency.

File: DP03_05_Near-Earth_Objects.ipynb (hunk 3 above)
Comment (sgreenstreet) [reply DP03_05_Near-Earth_Objects.ipynb:10]: Italics removed for consistency. And "CET Template Notebook" changed to "Orbital Properties of Near-Earth Objects (NEOs)".

File: DP03_05_Near-Earth_Objects.ipynb (hunk 4 above)
Comment (sgreenstreet) [reply DP03_05_Near-Earth_Objects.ipynb:86]: () moved for easier reading.

File: DP03_05_Near-Earth_Objects.ipynb (hunk 5 above)
Comment (sgreenstreet) [reply DP03_05_Near-Earth_Objects.ipynb:92]: () moved for easier reading as above.

File: DP03_05_Near-Earth_Objects.ipynb (hunk 6 above)
Comment (sgreenstreet) [reply DP03_05_Near-Earth_Objects.ipynb:444]: This query statement was copied from notebook DP03_04a_Introduction_to_Phase_Curves.ipynb for consistency and modified as needed. I can move the """ if needed.

File: DP03_05_Near-Earth_Objects.ipynb [hunk 8]
```diff
@@ -0,0 +1,1395 @@
...
+    "plt.title(\"NEO Class Boundaries\")\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ea0e5ba7-e504-4400-b002-348d601c3905",
+   "metadata": {},
+   "source": [
+    "To better understand the above ($a$, $e$) plot, remember that the Sun sits at the origin ($a$, $e$) = (0, 0) and the Earth has $a$ = 1.0 au and $e$ = 0.017, so it sits on the x-axis where the three blue curves converge. The space between the two outermost blue curves is Earth-crossing space where the Earth-crossing NEOs (Apollos and Atens) sit. A semimajor axis $a$ = 1.0 au divides the Earth-crossing NEOs into Apollos ($a$ > 1.0 au) and Atens ($a$ < 1.0 au). The black curve marks the $q$ < 1.3 au boundary for NEOs by definition. Amors sit between the $q$ < 1.3 au curve and the outer edge of Earth-crossing space (i.e., perihelia > Earth's aphelion = $q$ > 1.017 au); Amors thus always remain farther from the Sun than the Earth. Atiras are opposite to the Amors in that they always remain closer to the Sun than the Earth and live interior to Earth-crossing space; Atiras thus have aphelia < Earth's perihelion = $Q$ < 0.983 au."
```
[the remark this answers is not in the document]
Comment (sgreenstreet) [reply DP03_05_Near-Earth_Objects.ipynb:625]: "sit" changed to "is".

File: DP03_05_Near-Earth_Objects.ipynb (hunk 7 above)
Comment (sgreenstreet) [reply DP03_05_Near-Earth_Objects.ipynb:1355]: No longer in bold.

**Comment by sgreenstreet** [comment]
Made suggested changes and pushed to the branch.

**Review by ChristinaAdair — APPROVED** [review]
HI, looks good!

**Pull request merged**